messaging/handler.go

```diff
@@ -170,6 +178,68 @@ func (h *Handler) deleteReplica(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
+// subscribe creates a new subscription for a replica on a topic.
+func (h *Handler) subscribe(w http.ResponseWriter, r *http.Request) {
+	// Read the replica ID.
+	var replicaID uint64
+	if n, err := strconv.ParseUint(r.URL.Query().Get("replicaID"), 10, 64); err != nil {
+		h.error(w, ErrReplicaIDRequired, http.StatusBadRequest)
+		return
+	} else {
+		replicaID = uint64(n)
+	}
+
+	// Read the topic ID.
+	var topicID uint64
+	if n, err := strconv.ParseUint(r.URL.Query().Get("topicID"), 10, 64); err != nil {
+		h.error(w, ErrTopicRequired, http.StatusBadRequest)
+		return
+	} else {
+		topicID = uint64(n)
+	}
+
+	// Subscribe a replica to a topic.
+	if err := h.broker.Subscribe(replicaID, topicID); err == ErrReplicaNotFound {
```

Comment:
The topic ID is always valid. The replica will always subscribe to a topic -- even if a message hasn't been written to the topic yet.